libs/vue/src/builders/browser/schema.json

```diff
@@ -64,6 +64,10 @@
       "description": "Setting this to false can speed up production builds if you don't need source maps for production.",
       "default": false
     },
+    "transpileDependencies": {
+      "type": "array",
+      "description": "By default babel-loader ignores all files inside node_modules. If you want to explicitly transpile a dependency with Babel, you can list it in this option."
```

Comment:
Minor, but let's add `"default": []` so that it is inline with the vue cli.
https://cli.vuejs.org/config/#transpiledependencies

Reply:
done.

For testing it, I did in my project where it's needed. Here's the use case:
One of the dependencies uses a syntax not yet supported by default by Webpack (at least not the version that comes with vue-cli). In my case, that's https://github.com/ing-bank/lion/tree/master/packages/overlays that's uses some optional chaining